mp4parse/tests/public.rs

```diff
@@ -618,6 +618,24 @@ fn public_video_av1() {
     }
 }
 
+#[test]
+fn public_mp4_bug_1185230() {
```

Comment:
I wanted to do more here, but it was turning into a yak shave, so I've filed https://github.com/mozilla/mp4parse-rust/issues/264